caffe2/onnx/helper.h

```diff
@@ -16,24 +16,77 @@
 
 #pragma once
 
+#include "onnx/onnx_pb.h"
+
 #include <set>
 #include <string>
 #include <unordered_set>
 
-namespace caffe2 { namespace onnx {
+namespace caffe2 {
+namespace onnx {
+
+using ::ONNX_NAMESPACE::AttributeProto;
+using ::ONNX_NAMESPACE::NodeProto;
+
 class DummyName {
-  public:
-    static std::string NewDummyName();
+ public:
+  static std::string NewDummyName();
 
-    static void Reset(const std::unordered_set<std::string>& used_names);
+  static void Reset(const std::unordered_set<std::string>& used_names);
 
-    static void AddName(const std::string& new_used) {
-      get_used_names().insert(new_used);
-    }
+  static void AddName(const std::string& new_used) {
+    get_used_names().insert(new_used);
+  }
 
-   private:
-     static std::unordered_set<std::string>& get_used_names();
-     static size_t counter_;
+ private:
+  static std::unordered_set<std::string>& get_used_names();
+  static size_t counter_;
 };
 
-}}
+inline AttributeProto MakeAttribute(
```

Comment:
This functionality is similar to `utils/proto_utils.h`, albeit works with ONNX messages instead of Caffe2 messages. Maybe they could be unified?

Reply:
Let me check. I still prefer to have onnx part separated in case there is subtle mismatch between `caffe2::Argument` and `onnx::Attribute`.